README.md

```diff
@@ -1 +1,166 @@
+- Clone this repo to your local machine using `git clone https://github.com/ameerhamzakhalid/air-quality.git`
+- cd into the folder and run `npm install` to install all the dependencies.
+- Run `npm start` to start the app.
+- Open `http://localhost:3000` to view it in the browser.
+- The page will reload if you make edits.
+- You will also see any lint errors in the console.
+- Run `npm run build` to build the app for production to the `build` folder.
+- It correctly bundles React in production mode and optimizes the build for the best performance.
+- The build is minified and the filenames include the hashes.
+- Your app is ready to be deployed!
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Git and GitHub
+- A code editor
+- Nodejs
+- NPM
+- A browser
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Setup
+
+- Create a local directory where you can clone the project.
+- Clone the project into your directory
+- Install the dependencies
+- Open the project on a browser
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+
+Just run npm i to install all dependencies
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+**Ameer Hamza**
+
+- GitHub: [@githubhandle](https://github.com/ameerhamzakhalid)
+- Twitter: [@twitterhandle](https://twitter.com/@ameeerhamza1997)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/choudhary-hamza-37b17a141/)
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Improve the UI**
+- [ ] **Add functionality**
+- [ ] **Add the number of pages**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ameerhamzakhalid/air-quality/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project kindly leave a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank microverse for the great resources shared.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENCE) licensed.
```

Comment:
- [ ] Kindly fix the broken MIT license link in the readme file and add your own license.

Reply:
@omarsalem7 done sir!